### PR TITLE
Ws/add tests

### PR DESCRIPTION
**Pull request by spearw**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.csv filter=lfs diff=lfs merge=lfs -text
```
Comment (ralatsdc) [inline .gitattributes:1]: Um, I am sure this is correct, but, what does it do?

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+git lfs pull
```
Comment (ralatsdc) [inline test.sh:2]: Clever ... only takes a long time once. Nice solution.

**Review by ralatsdc — APPROVED** [review]
Please do respond to comments though ... for my education.

File: .gitattributes (hunk 1 above)
Comment (spearw) [reply .gitattributes:1]: It specifies which files to automatically store/retrieve as LFS, and a few other settings. This is the default recommended, and I specified storing all *.csv files.

**Pull request merged**